### ISSUE TITLE
Server crashes every couple of hours or so.

### ISSUE THREAD

**Issue by Convieo** [StrangeLoopGames/EcoIssues#3262]
Server crashes every couple of hours and creates the same dump file. I've already changes the compression level to "BestSpeed" to help, but the crashes seem still just as consistant.

`--BEGIN DUMP--
Dump Time
01/05/2018 11:50:13

Exception
System.IO.IOException: The process cannot access the file because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   at Ionic.Zip.ZipFile.Save()
   at Eco.Core.Serialization.DataStore.ProcessQueue()
   at Eco.Core.Serialization.DataStore.<Run>b__26_0()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()

--END DUMP--`

**Comment by MarjieAVolk** [comment]
ClayC says:

> When an item crashes the server, it writes to a dump file, good,
> When multiple instances of this item exist, and they all crash at the same time, they try to access the file but guess what, they crash again because they are all trying to access the same file at the same time.

**Comment by pamduncan** [comment]
I reported this issue some time ago - t was not related to multiple crashes but only happening once or twice a day but on multiple servers.  When we investigated it seemed to happen when the Game.eco was writing a save an trying to access the zip process.  We reduced the number of times it was happening by changing the compression speed in Storage.eco to 'best speed' and went from a few of these a day to a couple a week.  Meta thought it was because something else running on the machine was using that process  but we have 4 servers running now in isolation where the only thing running besides the OS processes are the game processes and there could not be anything else using the files/process and this still happens with about the same regularity it happens on other servers.
